src/EditorFeatures/Core/ExternalAccess/VSTypeScript/Api/VSTypeScriptFindUsagesContext.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.FindUsages;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.VSTypeScript.Api
+{
+    internal sealed class VSTypeScriptFindUsagesContext : IVSTypeScriptFindUsagesContext
```

Comment:
❔ Why is this in the API folder?

Reply:
It's returned by IVSTypeScriptStreamingFindUsagesPresenter.

Reply:
Shouldn't IVSTypeScriptStreamingFindUsagesPresenter be returning the interface `IVSTypeScriptFindUsagesContext` instead of the implementation?

Reply:
IVSTypeScriptFindUsagesContext does not have all methods that VSTypeScriptFindUsagesContext needs to expose.